client/app/cloud/project/compute/loadbalancer/configure/compute-loadbalancer-configure.html

```diff
@@ -0,0 +1,90 @@
+<div id="compute-loadbalancer-configure">
+    <h2><a ui-sref="^.loadbalancer"><i class="glyphicon glyphicon-chevron-left"></i></a> <span data-translate="cpc_loadbalancer_configure_title"></span></h2>
+    <div data-ng-if="CloudProjectComputeLoadbalancerConfigureCtrl.loaders.loadbalancer" class="text-center">
+        <oui-loader size="m" class="d-inline-block"></oui-loader>
+    </div>
+    <div data-ng-if="!CloudProjectComputeLoadbalancerConfigureCtrl.loaders.loadbalancer && CloudProjectComputeLoadbalancerConfigureCtrl.loadbalancer === null"
+    data-translate="cpc_loadbalancer_error">
+    </div>
+    <div data-ng-if="CloudProjectComputeLoadbalancerConfigureCtrl.toggle.updatedMessage && !CloudProjectComputeLoadbalancerConfigureCtrl.tasks.data.length" data-translate="cpc_loadbalancer_configure_updated_message"
+     class="oui-message"></div>
+
+    <div data-ng-if="CloudProjectComputeLoadbalancerConfigureCtrl.tasks.data.length" class="cui-advanced-options">
+        <h4 data-translate="cpc_loadbalancer_configure_task_waiting"></h4>
+        <div data-ng-repeat="task in CloudProjectComputeLoadbalancerConfigureCtrl.tasks.data track by $index" class="compute-loadbalancer-configure-tasks">
+            {{task.action}} : <cui-task-progress data-task="task"></cui-task-progress>
+        </div>
+    </div>
+    <form data-ng-if="!CloudProjectComputeLoadbalancerConfigureCtrl.loaders.loadbalancer && CloudProjectComputeLoadbalancerConfigureCtrl.loadbalancer !== null" data-ng-submit="CloudProjectComputeLoadbalancerConfigureCtrl.configure()">
+        <div class="cui-advanced-options" ng-if="CloudProjectComputeLoadbalancerConfigureCtrl.loadbalancer.status === 'custom'">
+            <a class="oui-button oui-button_secondary pull-right" data-translate="cpc_loadbalancer_configure_advanced" data-ui-sref="network.iplb.detail.frontends({'serviceName': CloudProjectComputeLoadbalancerConfigureCtrl.loadbalancerId})"></a>
+            <p>
+                <strong data-translate="cpc_loadbalancer_key_explain_1"></strong>
+            </p>
+            <p data-translate="cpc_loadbalancer_key_explain_2">
+            </p>
+        </div>
+        <div ng-if="CloudProjectComputeLoadbalancerConfigureCtrl.loadbalancer.status !== 'custom'">
+            <div><a class="oui-button oui-button_secondary pull-right" data-translate="cpc_loadbalancer_configure_advanced" data-ui-sref="network.iplb.detail.frontends({'serviceName': CloudProjectComputeLoadbalancerConfigureCtrl.loadbalancerId})"></a></div>
+            <h3 data-translate="cpc_loadbalancer_configure_loadbalancer"></h3>
+            <p class="oui-field ">
+                <span class="oui-label" data-translate="cpc_loadbalancer_configure_adresse"></span>
+                <span class="oui-label"><a ng-href="http://{{CloudProjectComputeLoadbalancerConfigureCtrl.loadbalancer.ipv4}}">{{CloudProjectComputeLoadbalancerConfigureCtrl.loadbalancer.ipv4}}</a></span>
```

Comment:
data-ng-href

Reply:
I don't have the guidelines, but I don't think we want to display an URL to the User